infrastructure/azure.tf

```diff
@@ -861,6 +876,8 @@ resource "azurerm_function_app" "azurerm_function_app_admin" {
     # Enable improved autoscaling algorithm
     # see https://www.azurefromthetrenches.com/azure-functions-significant-improvements-in-http-trigger-scaling/
     "WEBSITE_HTTPSCALEV2_ENABLED" = "1"
+
+    "StorageConnection" = "${azurerm_storage_account.azurerm_storage_account.primary_connection_string}"
```

Comment:
is this needed?

Reply:
Yes it is used here https://github.com/teamdigitale/io-functions-admin/blob/master/UpdateVisibleServicesActivity/index.ts#L7.